### ISSUE TITLE
Oddity in proportion mature

### ISSUE THREAD

**Issue by jysullivan** [commfish/seak_sablefish#24]
There appears to be a dip in the proportion mature at age-14. This function is from a logistic regression and should be a smooth line. Need to figure out if this is it a plotting artifact or what's going on.

![image](https://user-images.githubusercontent.com/7398155/54649382-d2786e80-4a5e-11e9-8a73-c30fdda41212.png)

**Comment by jysullivan** [comment]
I don't understand this one. Here is the underlying data:

![image](https://user-images.githubusercontent.com/7398155/54705317-5da24480-4af1-11e9-8a51-21fc5772610b.png)

There's nothing here that explains that dip for age-14, though I agree it's annoying.

---

To clarify these predictions are not directly from a logistic regression. When I compared age-based to length-based maturity curves, the length-based curve had better fit by AIC. I also looked at age by year and length by year combinations. The figure above represents a length-based logistic regression for maturity that was translated to age using the fitted curve for a length-based vonB from the same years. 

Here is the difference between the age-based (red) and the length-based (blue) that has been translated to age.  Light lines are annual mean predictions, dark lines are the means.
![image](https://user-images.githubusercontent.com/7398155/54705613-123c6600-4af2-11e9-8c04-c3f1d1d93350.png)

The AIC comparison:
![image](https://user-images.githubusercontent.com/7398155/54705872-91319e80-4af2-11e9-8558-df5ec175d0fa.png)
